src/commands/info/about.ts

```diff
@@ -10,15 +10,15 @@ import defaultEmbed, {
   DefaultEmbedType,
 } from '../../helpers/messaging/default-embed';
 import { t } from 'i18next';
-import XPError from '../../classes/xp-error';
+import XPError, { XPErrorType } from '../../classes/xp-error';
 
 const execute = async (interaction: ChatInputCommandInteraction) => {
   if (!interaction.guildId || !interaction.user.id || !interaction.guild) {
     await interaction.reply({
       content: 'Invalid interaction.',
       ephemeral: true,
     });
-    throw new XPError('Failed to send /about embed');
+    throw new XPError(XPErrorType.INTERACTION_GUILD_UNRESOLVABLE);
```

Comment:
This error is only for not being able to resolve a guild. Please add a separate condition for not being able to resolve the user id.